mopidy_musicbox_webclient/static/mb.manifest

```diff
@@ -1,6 +1,6 @@
 CACHE MANIFEST
 
-# 2016-01-31:v3
+# 2016-02-01:v1
 
 NETWORK:
 *
```

Comment:
We probably want to remove ``js/streamuris.js`` from the manifest file as well.

``printf "CACHE MANIFEST\n\n# 2016-MM-DD:vN\n\nNETWORK:\n*\n\nCACHE:\n" > mb.manifest; find . -type f | sed "s#^\./##" | grep -vi "ds_store" >> mb.manifest`` can also be run from the ``static`` directory to regenerate the entire manifest file in situations where that would be easier than updating it by hand.